tools/accuracy_checker/accuracy_checker/annotation_converters/squad.py

```diff
@@ -255,14 +255,15 @@ def convert(self, check_content=False, progress_callback=None, progress_interval
 
                 p_mask = self.fill_p_mask(span, len(truncated_query), sequence_added_tokens, cls_index)
                 idx = example_index
-                identifier = ['input_ids_{}'.format(idx), 'input_mask_{}'.format(idx), 'segment_ids_{}'.format(idx)]
+                identifier = ['input_ids_{}'.format(idx), 'input_mask_{}'.format(idx), 'segment_ids_{}'.format(idx), 'position_ids_{}'.format(idx)]
```

Comment:
please remove changes not connected with this model support

Reply:
it is connected to this model support

Reply:
removed